tests/util/test_config_reader.py

```diff
@@ -0,0 +1,54 @@
+import logging
+from typing import Generator
+import pytest
+
+from instana.util.config import parse_ignored_endpoints_from_yaml
+
+
+class TestConfigReader:
+    @pytest.fixture(autouse=True)
+    def _resource(self) -> Generator[None, None, None]:
+        yield
+
+    def test_load_file(self, caplog: pytest.LogCaptureFixture) -> None:
```

Comment:
I am not a fan of having a single test case with all possible cases written (or worse, duplicated) on it.
Please, create one test case using the parametrized feature of `pytest`, or two or more tests for each case.